packages/salesforcedx-vscode-soql/OSSREADME.json

```diff
@@ -0,0 +1,30 @@
+[
```

Comment:
You don't need this unless the code here has been copied from the vscode repository

Reply:
ok thanks, I'll remove the OSSREADME file.